app/modules/relationships/resources.py

```diff
@@ -0,0 +1,191 @@
+    """
+    Manipulations with Relationships.
+    """
+
+    @api.permission_required(
+        permissions.ModuleAccessPermission,
+        kwargs_on_request=lambda kwargs: {
+            'module': Relationship,
+            'action': AccessOperation.READ,
+        },
+    )
+    @api.login_required(oauth_scopes=['relationships:read'])
+    @api.parameters(PaginationParameters())
+    @api.response(schemas.BaseRelationshipSchema(many=True))
+    def get(self, args):
+        """
+        List of Relationship.
+
+        Returns a list of Relationship starting from ``offset`` limited by ``limit``
+        parameter.
+        """
+        return Relationship.query.offset(args['offset']).limit(args['limit'])
+
+    @api.permission_required(
+        permissions.ModuleAccessPermission,
+        kwargs_on_request=lambda kwargs: {
+            'module': Relationship,
+            'action': AccessOperation.WRITE,
+        },
+    )
+    @api.login_required(oauth_scopes=['relationships:write'])
+    @api.parameters(parameters.CreateRelationshipParameters())
+    @api.response(schemas.DetailedRelationshipSchema())
+    @api.response(code=HTTPStatus.CONFLICT)
+    def post(self, args):
+        """
+        Create a new instance of Relationship.
+        """
+
+        request_in = {}
+        import json
+
+        try:
+            request_in_ = json.loads(request.data)
+            request_in.update(request_in_)
+        except Exception:
+            pass
+
+        context = api.commit_or_abort(
+            db.session, default_error_message='Failed to create a new Relationship'
+        )
+
+        if (
+            request_in['individual_1_guid']
+            and request_in['individual_2_guid']
+            and request_in['individual_1_role']
+            and request_in['individual_2_role']
+        ):
+            with context:
+                relationship = Relationship(
+                    request_in['individual_1_guid'],
+                    request_in['individual_2_guid'],
+                    request_in['individual_1_role'],
+                    request_in['individual_2_role'],
+                )
+                db.session.add(relationship)
+                for member in relationship.individual_members:
+                    db.session.add(member)
+
+            rtn = {
+                'success': True,
+                'guid': relationship.guid,
+                'type': relationship.type,
+                'start_date': relationship.start_date,
+                'individual_members': [],
+            }
+            for member in relationship.individual_members:
+                each_member = {
+                    'individual_role': member.individual_role,
+                    'individual_guid': str(member.individual_guid),
+                }
+                rtn['individual_members'].append(each_member)
+            # ut.embed()
+            return rtn
+        else:
+            log.debug('FAILED TO CREATE RELATIONSHIP!')
```

Comment:
That's not a debug message, that'a a front end fault Audit log but to generate useful data about the reason you need better detection of what the fault is so that you can report it.